### ISSUE TITLE
Bookmark list layout broken if many labels

### ISSUE THREAD

**Issue by tuomas2** [AndBible/and-bible#2077]
![image](https://user-images.githubusercontent.com/5811789/151613651-c2d53723-ed4d-457f-9020-6511dc5c7e7f.png)

**Comment by agrogers** [comment]
Yes, and the labels are not that helpful unless you have a gift for
identifying colors!

It's on my to do list to rework that list and make it more useful.
